react/components/ProductAvailability.tsx

```diff
@@ -35,15 +51,11 @@ function ProductAvailability({
     )
   }
 
-  if (!highStockMessage) {
-    return null
+  if (!isLowStock && highStockMessage) {
+    return <HighStock text={highStockMessage} />
   }
 
-  return (
-    <Container>
-      <HighStock text={highStockMessage} />
-    </Container>
-  )
+  return null
```

Comment:
This changes the behavior of the component. if `isLowStock` is `true` and `highStockMessage` is also `true` then now `HighStock` will not be rendered. Although this makes sense, this breaks the previous behavior, and we can't support that unless we release a new major.

Also, we always make the conditions return the exceptional cases, such as `null`. The default or "intended" behavior is to return a component, so it should be the last return statement of the function.